### PR TITLE
Adds generic label, as per new commerceguys/tax changes.

### PR DESCRIPTION
**Pull request by joshuataylor**
This allows
https://github.com/commerceguys/tax/commit/6ab2f4cbef41c528b2098fa56140f68bf676c001
to work.

### CODE REVIEW

File: modules/tax/src/Entity/TaxType.php [hunk 1]
```diff
@@ -229,4 +236,18 @@ public function hasRate(TaxRateInterface $rate) {
...
+   */
+  public function getGenericLabel() {
+    return $this->genericLabel;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setGenericLabel($genericLabel) {
+    $this->genericLabel = $genericLabel;
```
Comment (bojanz) [inline modules/tax/src/Entity/TaxType.php:252]: You're missing the GenericLabel::assertExists($genericLabel); line in this method.

Comment (joshuataylor) [inline modules/tax/src/Entity/TaxType.php:252]: Fixed

File: modules/tax/src/Entity/TaxType.php [hunk 2]
```diff
@@ -44,7 +45,6 @@
  * )
  */
 class TaxType extends ConfigEntityBase implements TaxTypeInterface {
-
```
Comment (bojanz) [inline modules/tax/src/Entity/TaxType.php:47]: D8 coding standards impose a newline here, and at the end.

Comment (joshuataylor) [inline modules/tax/src/Entity/TaxType.php:47]: Fixed

**Pull request merged**